packages/embark-blockchain-process/src/blockchain.js

```diff
@@ -216,7 +213,19 @@ Blockchain.prototype.run = function () {
         next();
       });
     },
-    function init(next) {
+    function initPluginClient(next) {
+      if (!self.isCustomPlugin) {
+        return next();
+      }
+      if (self.client.initChain) {
```

Comment:
Again, that `if` is not really useful, because the function will always be defined in the parent class

Reply:
True, this should probably be removed.